portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+    <head>
```

Comment:
Should be indented two spaces rather than 4!

Reply:
Sorry about the indentation :( I'll be more careful about it in the future commits.